### PR TITLE
suil 0.8.2 (new formula)

### PR DESCRIPTION
**Pull request by david0**

### CODE REVIEW

File: Library/Formula/suil.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <suil/suil.h>
+
+      int main()
+      {
+        return suil_ui_supported("my-host", "my-ui");
+      }
+    EOS
+    args = `pkg-config suil-0 --cflags --libs`.split
```
Comment (bfontaine) [inline Library/Formula/suil.rb]: You can’t rely on `pkg-config` here since it’s only a build dependency.

Comment (david0) [inline Library/Formula/suil.rb]: ok.. fixed.

**Comment by david0** [comment]
I fixed the problem, could you have a look at the formula again?

File: Library/Formula/suil.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      #include <suil/suil.h>
+
+      int main()
+      {
+        return suil_ui_supported("my-host", "my-ui");
+      }
+    EOS
+    args = ["-I#{include}/suil-0", "-L#{lib}", "-lsuil-0"]
+    args += %w[test.c -o test]
+    system ENV.cc, *args
```
Comment (bfontaine) [inline Library/Formula/suil.rb]: You can inline all arguments here: `system ENV.cc, "-I#{include}/suil-0", "-L#{lib}", "-lsuil-0", "test.c", "-o", "test"`.

**Comment by david0** [comment]
I updated the formula once again, please check.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @david0; we appreciate it! :tada:

**Pull request closed without merging**